llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -274,6 +276,10 @@ static Instruction *getInstructionForCost(const VPRecipeBase *R) {
     return IG->getInsertPos();
   if (auto *WidenMem = dyn_cast<VPWidenMemoryRecipe>(R))
     return &WidenMem->getIngredient();
+  // FIXME: Override the cost method properly to take gather/scatter cost
```

Comment:
Histogram cost computation looks self contained, can we instead directly implement `VPHistogramRecipe::computeCost` instead falling back?

Reply:
done

Reply:
Thanks, can the FIXME & histogram case below be removed?

Reply:
They were removed. Are you looking at a cached version of the diff?